dockertls/README.md

```diff
@@ -63,6 +63,34 @@ $env:DOCKER_TLS_VERIFY="1"
 docker version
 ```
 
+## Managing Multiple Hosts
+
+## First Host
+For the first host you must create an empty directory to hold the Certificate Authority Private Keys.
+Ideally the key and password should be kept separate and only provided when additional certificates are created.
+```
+mkdir $env:SystemDrive\DockerSSLCARoot
+mkdir $env:USERPROFILE\.docker
+docker run --rm `
+  -e SERVER_NAME=$(hostname) `
+  -e IP_ADDRESSES=127.0.0.1,192.168.254.135 `
```

Comment:
What is this second IP used for? Does someone need to adjust it for their environment?